### ISSUE TITLE
Better Documentation for Contributors?

### ISSUE THREAD

**Issue by Leonardo-Ferreira** [motiv-labs/janus#475]
I'm new here and I'd like to contribute and find the documentation "lacking". It would be nice to have a basic walkthrough for a developer, something like: 
clone the code, then copy the file `janus/devHelpers/Janus` and place it in `/etc/Janus/`, put that other file on that other location and run the code with the flag "start". You will be able to call `http://localhost/echo` and get "echo" back.

you know? first steps to get people compiling and debugging real quick, no fuss...

**Comment by jtesser** [comment]
Hi @Leonardo-Ferreira 

We would welcome contribution. Our current gitbook is published here https://motivlabs.gitbook.io/janus/
We do need some better quick start stuff and examples for sure.

**Comment by Leonardo-Ferreira** [comment]
yeah... took me while to get things running... now im trying to retrace my steps in order to make a guide
